packages/lists/src/components/Viewport/Viewport.tsx

```diff
@@ -87,12 +87,18 @@ export const Viewport = (props: IViewportProps): JSX.Element => {
 
       return () => {
         if (scrollContainerRef.current) {
+          // TODO: check with @soettl on what to do in response to this warning:
```

Comment:
@soettl How would you recommend handling this? Just disabling the rule for this area is an option, but if there's a "right" way to fix it that's better.